src/tests/ftest/recovery/pass_2.py

```diff
@@ -0,0 +1,211 @@
+        another engine where we didn’t create.
+        6. Enable and start the checker.
+        7. Query the checker and verify that the issue was fixed.
+        i.e., Current status is COMPLETED.
+        8. Disable the checker.
+        9. Restart the servers so that the storage usage in the next step is updated.
+        10. Call dmg storage query usage to verify that the pool usage is back to the
+        original value.
+
+        Jira ID: DAOS-11734
+
+        :avocado: tags=all,pr
+        :avocado: tags=hw,medium
+        :avocado: tags=recovery,pass_2
+        :avocado: tags=Pass2Test,test_orphan_pool_shard
+        """
+        # 1. Create a pool.
+        self.pool = self.get_pool(connect=False, target_list="0")
+
+        # 2. Call dmg storage query usage to store the default space utilization.
+        rank_to_free_orig = self.get_rank_to_free()
+        self.log.info("rank_to_free_orig = %s", rank_to_free_orig)
+
+        # 3. Prepare to copy the pool path before stopping servers.
+
+        # In order to copy the pool directory without password, there are several things
+        # to determine and set up.
+
+        # 3-1. Determine source host and destination host. Source host is where rank 0 is.
+        # Destination host is the other host.
+        rank_to_host = self.server_managers[0].ranks
+        self.log.info("rank_to_host = %s", rank_to_host)
+        src_host = NodeSet(rank_to_host[0])
+        dst_host = None
+        for rank in range(1, 4):
+            if rank_to_host[rank] != str(src_host):
+                dst_host = NodeSet(rank_to_host[rank])
+                break
+        self.log.info("src_host = %s; dst_host = %s", src_host, dst_host)
+
+        # 3-2. Determine source and destination mount point. First, find the source mount
+        # point that maps to rank 0. Then use the same mount point for the destination.
+        # This way, we can handle the case where the mount point name is changed in the
+        # future. At the same time, determine the destination rank, which is where
+        # dst_mount is mapped.
+        # dst_mount = "/mnt/daos0"
+        src_mount = None
+        dst_mount = None
+        dst_rank = None
+        host_list = str(self.hostlist_servers)
+        dmg_command = self.get_dmg_command()
+        storage_query_out = dmg_command.storage_query_usage(
+            host_list=host_list)
+        hash_dict = storage_query_out["response"]["HostStorage"]
+        for storage_dict in hash_dict.values():
+            if str(src_host) in storage_dict["hosts"]:
+                # Determine source mount point that maps to rank 0.
+                for scm_namespace in storage_dict["storage"]["scm_namespaces"]:
+                    if scm_namespace["mount"]["rank"] == 0:
+                        src_mount = scm_namespace["mount"]["path"]
+                        # Use the same mount point as source.
+                        dst_mount = src_mount
```

Comment:
Just FYI if you prefer
```suggestion
                        # Use the same mount point as source.
                        dst_mount = src_mount = scm_namespace["mount"]["path"]
```

Reply:
Done